nanoFramework.System.Net.Http/Http/StringContent.cs

```diff
@@ -0,0 +1,93 @@
+﻿//
+// Copyright (c) .NET Foundation and Contributors
+// Portions Copyright (c) Microsoft Corporation.  All rights reserved.
+// See LICENSE file in the project root for full license information.
+//
+
+using System.Net.Http.Http.Headers;
+using System.Text;
+
+namespace System.Net.Http
+{
+    /// <summary>
+    /// Provides HTTP content based on a string.
+    /// </summary>
+    public class StringContent : ByteArrayContent
+    {
+        private const string _DefaultMediaType = "text/plain";
```

Comment:
if it's a constant, then no need of the _ before the name of the variable. Same for the next one